### PR TITLE
ROC 4838 update court calculation with 2019 2020 allowances

### PR DESCRIPTION
**Pull request by tharacka**
### JIRA link

https://tools.hmcts.net/jira/browse/ROC-4838

### Change description

Add 2019/2020 allowance changes to the json and make it so that it goes live on the 05/04/19

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [ ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by andrewwa-kainos** [comment]
This solution will need a system restart for the allowance transition to take effect

File: src/main/app/common/allowances/allowance.ts [hunk 1]
```diff
@@ -24,22 +26,49 @@ export enum LivingAllowanceType {
...
+  constructor (public allowance?: Allowance[]) {
+  }
+
+  deserialize (input?: any): Allowance {
+    if (!input) {
+      return input
+    }
+    this.allowance = this.deserializeRows(input.allowances)
+    return this.allowance
+      .sort((a, b) => {
```
Comment (andrewwa-kainos) [inline src/main/app/common/allowances/allowance.ts]: Can be something like...
`.sort((a, b) => a.startDate.diff(b.startDate, 'days'))`
... for readability

File: src/test/app/common/statement-of-means/statementOfMeansCalculations.ts [hunk 2]
```diff
@@ -521,12 +521,72 @@ describe('StatementOfMeansCalculations', () => {
...
+              pensioner : [ { item: 'item3', weekly: 10, monthly: 50 } ],
+              disability : [ { item: 'item4', weekly: 10, monthly: 50 } ],
+              startDate : moment().add(1, 'day').toISOString()
+            }]
+          }
+          const allowance: Allowance = new Allowances().deserialize(input)
+          expect(allowance).to.equal(undefined)
+        })
+      })
+      context('when there is multiple data present', () => {
```
Comment (andrewwa-kainos) [inline src/test/app/common/statement-of-means/statementOfMeansCalculations.ts]: minor, grammar. data is plural
'when there are multiple data present'

File: src/main/app/common/allowances/allowance.ts [hunk 3]
```diff
@@ -24,22 +26,50 @@ export enum LivingAllowanceType {
...
+        // return a.startDate.toDate().getTime() - b.startDate.toDate().getTime()
```
Comment (robertparkinson) [inline src/main/app/common/allowances/allowance.ts]: commented code to be removed

**Pull request merged**